test/factories/internship_offers.rb

```diff
@@ -28,6 +29,7 @@
     sequence(:title) { |n| "Stage de 3è - #{n}" }
     description { 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin eros orci, iaculis ut suscipit non, imperdiet non libero. Proin tristique metus purus, nec porttitor quam iaculis sed. Aenean mattis a urna in vehicula. Morbi leo massa, maximus eu consectetur a, convallis nec purus. Praesent ut erat elit. In eleifend dictum est eget molestie. Donec varius rhoncus neque, sed porttitor tortor aliquet at. Ut imperdiet nulla nisi, eget ultrices libero semper eu.' }
     sector { create(:sector) }
+    school_type { InternshipOffer.school_types.keys.sample.to_sym }
```

Comment:
```suggestion
    school_type { InternshipOffer.school_types.keys.first.to_sym }
```
eviter le random sur les tests, si derrieredes comportements changent on va galerer a comprendre prk certains tests cassent...